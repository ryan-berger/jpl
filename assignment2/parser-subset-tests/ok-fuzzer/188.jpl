time time let I = v(2., .0, 30)
assert .5731, ""
