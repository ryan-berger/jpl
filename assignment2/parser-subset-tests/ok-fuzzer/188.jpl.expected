(TimeCmd
 (TimeCmd
  (StmtCmd
   (LetStmt
    (ArgLValue (VarArgument I))
    (CallExpr v (FloatExpr 2) (FloatExpr 0) (IntExpr 30))))))
(StmtCmd (AssertStmt (FloatExpr 0) ""))
